core/domain/topic_services.py

```diff
@@ -267,8 +271,21 @@ def apply_change_list(topic_id, change_list):
                       topic_domain.TOPIC_PROPERTY_LANGUAGE_CODE):
                     topic.update_language_code(change.new_value)
                 elif (change.property_name ==
-                      topic_domain.TOPIC_PROPERTY_THUMBNAIL_FILENAME):
-                    topic.update_thumbnail_filename(change.new_value)
+                      topic_domain.TOPIC_PROPERTY_THUMBNAIL_FILENAME_AND_SIZE_IN_BYTES): # pylint: disable=line-too-long
+                    try:
+                        fs = fs_domain.AbstractFileSystem(
+                            fs_domain.GcsFileSystem(
+                                feconf.ENTITY_TYPE_TOPIC, topic_id))
+                        thumbnail_size_in_bytes = len(fs.get(
+                            '%s/%s' % (
+                                ASSET_TYPE_THUMBNAIL,
```

Comment:
Why do we need to introduce a new constant here? There should be some common logic shared between this and whatever process in the backend is currently storing the thumbnail in the datastore, and any shared constants should be used in those other places too.

Reply:
Procedure for saving the thumbnail in backend is [here](https://github.com/oppia/oppia/blob/9b0478905fc8037c23536546af7b6819a805d498/core/controllers/editor.py#L673).
Updated the changes accordingly.